service/src/test/java/bio/terra/landingzone/stairway/flight/create/CreateLandingZoneResourcesFlightIntegrationTest.java

```diff
@@ -0,0 +1,142 @@
+package bio.terra.landingzone.stairway.flight.create;
+
+import static org.awaitility.Awaitility.await;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+
+import bio.terra.common.iam.BearerToken;
+import bio.terra.common.stairway.StairwayComponent;
+import bio.terra.landingzone.job.exception.InternalStairwayException;
+import bio.terra.landingzone.job.exception.JobNotFoundException;
+import bio.terra.landingzone.library.landingzones.LandingZoneTestFixture;
+import bio.terra.landingzone.library.landingzones.management.LandingZoneManager;
+import bio.terra.landingzone.service.landingzone.azure.LandingZoneService;
+import bio.terra.landingzone.service.landingzone.azure.model.LandingZoneRequest;
+import bio.terra.landingzone.stairway.flight.StepsDefinitionFactoryType;
+import bio.terra.landingzone.stairway.flight.create.resource.step.AggregateLandingZoneResourcesStep;
+import bio.terra.profile.model.CloudPlatform;
+import bio.terra.profile.model.ProfileModel;
+import bio.terra.stairway.FlightState;
+import bio.terra.stairway.FlightStatus;
+import bio.terra.stairway.exception.FlightNotFoundException;
+import bio.terra.stairway.exception.StairwayException;
+import java.time.Duration;
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.retry.annotation.EnableRetry;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+@Tag("integration")
+@ActiveProfiles("test")
+@PropertySource("classpath:integration_azure_env.properties")
```

Comment:
This tells spring to look for a an additional file, in this case the one generated by `render-test-config.sh`